packages/gatsby-theme-docs/bin/create-release-note.js

```diff
@@ -4,68 +4,112 @@ const fs = require('fs');
+        },
+        {
+          title: 'fix',
+          value: 'fix',
+          description: 'for bug fixes',
+        },
+        {
+          title: 'announcement',
+          value: 'announcement',
+          description:
+            'for release notes with no product change nor immediate product change',
+        },
+      ],
+    },
+    {
+      type: 'text',
+      name: 'date',
+      message: 'Insert the release date with the following format: YYYY-MM-DD',
+      validate: (date) => {
+        const checkDateFormat = moment(date, 'YYYY-MM-DD', true);
+        return checkDateFormat.isValid() ? true : 'Please use a valid format!';
       },
-      {
-        title: 'feature',
-        value: 'feature',
-        description: 'for the introduction of an entirely new feature',
-      },
-      {
-        title: 'fix',
-        value: 'fix',
-        description: 'for bug fixes',
-      },
-      {
-        title: 'announcement',
-        value: 'announcement',
-        description:
-          'for release notes with no product change nor immediate product change',
-      },
-    ],
-  },
-  {
-    type: 'date',
-    name: 'date',
-    message: 'Pick a release date',
-  },
-  {
-    type: 'text',
-    name: 'title',
-    message: 'The title of the release note. Max 256 character plain text.',
-    validate: (title) =>
-      title.length > 256 ? 'Title is longer than 256 chars!' : true,
-  },
-  {
-    type: 'text',
-    name: 'description',
-    message:
-      'Key changes and their advantages. Max 256 character plain text. This text is tweet-able and used in RSS feeds.',
-    format: (val, values) => `${values.title} ${val}`,
-    validate: (description) =>
-      description.length > 256 ? 'Description is longer than 256 chars!' : true,
-  },
-  {
-    type: 'list',
-    name: 'topics',
-    message:
-      'Write down the topics for this release note. Seperate them with commas.',
-    separator: ',',
-  },
-];
+    },
+    {
+      type: 'text',
+      name: 'title',
+      message: 'Insert the title for the release note. Write it in plain text.',
+      validate: (title) =>
+        title.length > 256 ? 'Title is longer than 256 chars!' : true,
+    },
+    {
+      type: 'text',
+      name: 'description',
+      message:
+        'Describe the key changes and their advantages briefly. Max 256 character plain text. This text is tweet-able and used in RSS feeds.',
+      validate: (description) =>
+        description.length > 256
+          ? 'Description is longer than 256 chars!'
+          : true,
+    },
+    customizedQuestions,
+  ];
+};
 
 (async () => {
-  const response = await prompts(questions);
+  const folderpath = path.join(process.cwd(), `src/releases`);
+  if (!fs.existsSync(folderpath)) {
+    console.error(`No "releases" folder found in this directory`);
+    process.exit();
+  }
+
+  const getQuestions = () => {
+    const moduleName = 'release-note-topic-config';
```

Comment:
I would keep this config file "generic" to the script, and not specific to the topics. The topics is then an option within the config.

```suggestion
    const moduleName = 'docs-release-notes-config';
```

Reply:
Hmm okay. Yes I thought about that, too. Thought it's more clear that way because it suggests that you can only configure topics, nothing else. But that's fine for me.